contracts-periphery/package.json

```diff
@@ -2,11 +2,20 @@
   "name": "@umbra/contracts-periphery",
   "version": "0.0.1",
   "license": "MIT",
+  "files": [
+    "typechain"
+  ],
   "scripts": {
     "install": "make install",
-    "build": "make build",
+    "build": "make build && yarn run typechain --target ethers-v5 --out-dir ./typechain './out/UmbraBatchSend.sol/UmbraBatchSend.json'",
     "test": "make test",
     "lint": "forge fmt --check",
-    "coverage": "make coverage"
+    "coverage": "make coverage",
+    "prepare": "yarn run typechain --target ethers-v5 --out-dir ./typechain './out/UmbraBatchSend.sol/UmbraBatchSend.json'"
```

Comment:
If we do postpone typechain integration, let's delete all the new typechain stuff added to this file

Reply:
Removed https://github.com/ScopeLift/umbra-protocol/pull/426/commits/b68763f63027e9a50cb4e4d4f4c112a7d72268fe